require.php

```diff
@@ -44,6 +44,10 @@
 	require_once WP_FS__DIR_INCLUDES . '/entities/class-fs-payment.php';
 	require_once WP_FS__DIR_INCLUDES . '/entities/class-fs-plugin-license.php';
 	require_once WP_FS__DIR_INCLUDES . '/entities/class-fs-subscription.php';
+    /**
+     * Preemptively load the latest version of `Freemius_Api_WordPress` class so that other non-standard integration and custom forks do not override it. For non-standard or custom forks, we recommend copying the source code and renaming the classes, or putting them under namespaces.
+     */
+    require_once WP_FS__DIR_SDK . '/FreemiusWordPress.php';
```

Comment:
@drory82 In case you're referring to spaces, we have started to transition from tabs to spaces since a long time ago. However, if you meant something else, would you please clarify?